skrf/tests/test_network.py

```diff
@@ -1743,6 +1743,28 @@ def test_stability(self):
         with pytest.raises(ValueError):
             net.stability
 
+    def test_equality(self):
+        s = npy.random.randn(10, 2, 2)
+        f1 = npy.arange(10)
+        f2 = npy.arange(10)
+        n1 = rf.Network(s=s,f=f1)
+        n2 = rf.Network(s=s,f=f2)
+        self.assertTrue(n1 == n2)
+
+        f2 = npy.arange(11)
+        n2 = rf.Network(s=s,f=f2)
+        self.assertFalse(n1 == n2)
```

Comment:
Maybe one more test with f2 = f1 * 10, as this test returns when checking the length of the vectors.